docs/release_notes/next/dev-2035-process-count

```diff
@@ -0,0 +1 @@
+#2035: Add setting for number of processes to start in multiprocessing pool
```

Comment:
NITPICK: probably best to specify the number of processes we are capping startup too uin the release notes so it's easy to find if searching for this in the future.